pallets/ah-migrator/src/lib.rs

```diff
@@ -478,9 +488,11 @@ pub mod pallet {
 		) -> DispatchResult {
 			ensure_root(origin)?;
 
-			Self::do_receive_accounts(accounts)?;
+			let res = Self::do_receive_accounts(accounts);
 
-			Ok(())
+			Self::increment_msg_received_count(res.is_err());
```

Comment:
Ah I see. I thought about having a wrapper extrinsic that does this, but also fine like this.